bootstrap/proxy/service.tf

```diff
@@ -17,11 +19,51 @@ resource "kubernetes_service_v1" "proxy_service" {
 
     port {
       name        = "proxy"
-      port        = 443
+      port        = 9443
```

Comment:
Is there a reason for changing the load balancer port and changing the protocol? this will work our existing infra as is I believe

Reply:
I was trying to remember if it was related to cert-manager, but I changed it back to 433 and HTTPS because I have no valuable reason